runtime/tests/runtime_integration_tests/test_ecdsa.rs

```diff
@@ -248,3 +248,32 @@ fn test_ecdsa_verify_bad_chksum() {
         resp,
     );
 }
+
+// HW errors are not supported on the SW emulator yet
+#[cfg(any(feature = "verilator", feature = "fpga_realtime"))]
```

Comment:
nit: if there isn't one already, could you open an issue for emulator support?

Reply:
No problem, done https://github.com/chipsalliance/caliptra-sw/issues/1615